src/apps/vpn/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -20,10 +20,6 @@ ColumnLayout {
         State {
             when: type === "text"
 
-            PropertyChanges {
```

Comment:
Curious: Do we know if there is a bug filed with Qt for this?

Reply:
Couldn't find anything - but appears to be resolved in 6.5